inc/codeception/overrides.php

```diff
@@ -7,10 +7,9 @@
 
 // phpcs:disable PSR1.Files.SideEffects
 
-
-// Flush cache now, and register shutdown event to clear the cache, also triggers if an error occurs.
+// Register shutdown event to clear the cache, also triggers if an error occurs.
 register_shutdown_function( function () {
-	if ( function_exists( 'wp_cache_flush' ) ) {
+	if ( ( ! defined( 'WP_BROWSER_TEST' ) || ! WP_BROWSER_TEST ) && function_exists( 'wp_cache_flush' ) ) {
```

Comment:
Can you explain what the checks here does in a comment please ?

Reply:
Yep, also thinking I should make that constant name more specific 🤔

Reply:
Actually no it's ok I think - same approach applies for acceptance and functional tests.